packages/mobile/e2e/src/WalletConnect.spec.js

```diff
@@ -7,6 +7,7 @@ describe('Given Wallet Connect', () => {
     await quickOnboarding()
   })
 
+  // Testing to see if walletConnectV1 is the issue
```

Comment:
is this comment necessary?

Reply:
Removed in https://github.com/valora-inc/wallet/pull/1837/commits/7609f7d602b2cf6e67e6b41843c51bd3d993a984!